_data-prepper/managing-data-prepper/configuring-data-prepper.md

```diff
@@ -100,3 +101,102 @@ usage | Yes | Bytes | Specifies the JVM heap usage at which to trip a circuit br
+
+Configures [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) extension plugin to be 
+referenced in pipeline plugin configurations, e.g.
+
+```
+extensions:
+  aws:
+    secrets:
+      <YOUR_SECRET_CONFIG_ID_1>:
+        secret_id: <YOUR_SECRET_ID_1>
+        region: <YOUR_REGION_1>
+        sts_role_arn: <YOUR_STS_ROLE_ARN_1>
+        refresh_interval: <YOUR_REFRESH_INTERVAL>
+      <YOUR_SECRET_CONFIG_ID_2>:
+        ...
+```
+
+| Option  | Required | Type   | Description                                                                                  |
+|:--------|:---------|:-------|:---------------------------------------------------------------------------------------------|
+| secrets | No       | Object | The AWS Secrets Manager extension plugin configuration. See [Secrets](#secrets) for details. |
+
+###### Secrets
+
+Multiple secrets configuration objects can be defined with unique id for each.
+
+| Option           | Required | Type     | Description                                                                                                                                                                                                                                                    |
+|:-----------------|:---------|:---------|:---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| secret_id        | Yes      | String   | The AWS secret name or ARN.                                                                                                                                                                                                                                    |
+| region           | No       | String   | The AWS region of the secret. Defaults to `us-east-1`.                                                                                                                                                                                                         |
+| sts_role_arn     | No       | String   | The AWS Security Token Service (AWS STS) role to assume for requests to AWS Secrets Manager. Defaults to `null`, which will use the [standard SDK behavior for credentials](https://docs.aws.amazon.com/sdk-for-java/latest/developer-guide/credentials.html). |
+| refresh_interval | No       | Duration | The refreshment interval for AWS secrets extension plugin to poll new secret values. See [Secrets refreshment](#secrets-refreshment) for details. Defaults to `PT1H`.                                                                                          |
+
+###### Reference secrets
+
+In `pipelines.yaml`, secret values can be referenced within pipeline plugins using the following formats:
+
+* plaintext: `${{aws_secrets:<YOUR_SECRET_CONFIG_ID>}}`.
```

Comment:
We should be clearly state in the preceding paragraph that users need to replace <YOUR_SECRET_CONFIG_ID> including the <>.